README.md

```diff
@@ -311,6 +312,7 @@ Besides providing easy-to-integrate and scalable blockchain APIs, Infura empower
 1. GetBlock - [https://getblock.io/nodes/algo/](https://getblock.io/nodes/algo/)
 2. Pocket Network - [https://mainnet.portal.pokt.network](https://mainnet.portal.pokt.network)
 3. NOWNodes.io - [https://nownodes.io/](https://nownodes.io/)
+4. Alchemy - [https://alchemy.com](https://docs.alchemy.com/reference/polygon-zkevm-api-quickstart)
```

Comment:
Ser I think you added it under `Algorand` not `Polygon zkEVM`

Reply:
Ahh 🤦